prometheus-podman-exporter.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: prometheus-podman-exporter
+  version: 1.13.3
+  epoch: 0
+  description: Prometheus exporter for podman environments exposing containers, pods, images, volumes and networks information.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - btrfs-progs-dev
+      - gpgme-dev
+  environment:
+    CGO_ENABLED: "0"
```

Comment:
required else we're running into build failure finally leading to gcc bugzilla forum.